docs(tutorials): update traefik tuto (#1635)
scaleway/docs-content: tutorials/traefik-v2-cert-manager/index.mdx

<Lightbox src="scaleway-traefik_widlcard.webp" alt="" />

<Message type="tip">

>   Traefik listens by default on port 80 and 443 of the public IP of the node on which it is deployed. You can retrieve the external IP of your clusters' nodes by runing the `kubectl get node -o wide` command.

Author: ```suggestion
  Traefik listens by default on ports 80 and 443 of the public IP of any node of your cluster. You can retrieve the external IP of your clusters' nodes by running the `kubectl get node -o wide` command.
```
